.github/workflows/automated-pr.yml

```diff
@@ -0,0 +1,19 @@
+name: Dependabot Pull Request
+on:
+  schedule:
+  - cron: "0 16 * * 4"
```

Comment:
Why not do this earlier in the week before release day?

Reply:
I didn't want us to have a lot of PRs to review, but it does make sense to change this to Monday.